users/views.py

```diff
@@ -4,6 +4,7 @@
 from django.http import Http404
 from django.contrib.auth.forms import PasswordChangeForm
 from django.contrib.auth import update_session_auth_hash
+from django.utils.safestring import mark_safe
```

Comment:
mark_safe is here too. Best to include it if we can, but if you can determine it's causing the issues, go ahead and remove.